lumber.js

```diff
@@ -11,4 +11,6 @@ const packagejson = require('./package.json');
 program
   .version(packagejson.version)
   .command('generate <appName>', 'generate a backend application with an ORM/ODM configured')
+  .command('login', 'login onto foreastadmin api')
+  .command('logout', 'log out from forestadmin api')
```

Comment:
It seems it's a new feature (or at least a new implementation of an undocumented feature). If so, I think it should contain a new line in changelog. I feel like we also could ask to @marcperrinp for his review: there are 2 new texts, I'm not totally confident about the wording (example: `log in onto`, `forestadmin` in one word and `api` lowercase).

Still I have to test it now! 😅

Reply:
line 14 --> log into forestadmin api (FYI https://grammarist.com/spelling/log-in-login/ so it should be "log in" not "login"; however, here "in" becomes "into", therefore "log into".

line 15 is good 💯

Reply:
`api` vs `API`: it seems it should be upper case: 
 - https://en.wikipedia.org/wiki/Application_programming_interface
- https://docs.forestadmin.com/documentation/reference-guide/how-it-works/admin-back-end

`forestadmin` vs `Forest Admin`: maybe I'm wrong, but it seems it's `Forest Admin` in documentation.

OK @marcperrinp & @VincentMolinie ?

Reply:
yeah totally right